app/concepts/reservations/use_cases/cancel_unpaid_reservations.rb

```diff
@@ -9,7 +9,11 @@ def initialize(repository: Reservations::Repository.new)
       def call(screening_id)
           reservations = @repository.unpaid(screening_id)
           reservations.map do |reservation|
+            client = Clients::UseCases::FindBy.new.call(id: reservation.client_id)
```

Comment:
Why not use `Clients::Repository.new.find_by(reservation.client_id)`?

Reply:
Yes. You are right there is no need for UseCase for that :). I am still a little confused with UseCases and Repositories.

Reply:
>I am still a little confused with UseCases and Repositories.

The boundary between one and another is quite easy - if your endpoint doing more than one thing, you should create use case. If there is only one functionality, like one-liner to fetch some records - you can use just repository.